OpenRA.Mods.Common/Traits/World/ScriptLobbyDropdown.cs

```diff
@@ -0,0 +1,66 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2016 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+using System.Linq;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Controls the map difficulty, tech level, and short game lobby options.")]
+	public class ScriptLobbyDropdownInfo : ITraitInfo, ILobbyOptions
+	{
+		[FieldLoader.Require]
+		[Desc("Internal id for this option.")]
+		public readonly string ID = null;
+
+		[FieldLoader.Require]
+		[Desc("Display name for this option.")]
+		public readonly string Label = null;
+
+		[FieldLoader.Require]
+		[Desc("Default value key.")]
```

Comment:
"Key pointing to the default value"? Default value alone can indeed be misunderstood.